src/ui/public/vislib/visualizations/point_series/line_chart.js

```diff
@@ -169,7 +171,9 @@ export default function LineChartFactory(Private) {
           .interpolate(interpolate)
           .x(isHorizontal ? cx : cy)
           .y(isHorizontal ? cy : cx);
-        return d3Line(data.values);
+        return d3Line(data.values.filter(function (d) {
```

Comment:
given that we use this so often, perhaps move the `isNull` predicate to a separate util-module? (e.g. as lodash mixin).